tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -210,23 +212,24 @@
                 InternalApi: internal_api
                 Storage: storage
                 Tenant: tenant
+
               # edpm_nodes_validation
               edpm_nodes_validation_validate_controllers_icmp: false
               edpm_nodes_validation_validate_gateway_icmp: false
 
-              edpm_ovn_metadata_agent_DEFAULT_transport_url: rabbit://default_user_-secret@rabbitmq.openstack.svc:5672
-              edpm_ovn_metadata_agent_metadata_agent_ovn_ovn_sb_connection: tcp:172.17.0.31:6642
-              edpm_ovn_metadata_agent_metadata_agent_DEFAULT_nova_metadata_host: 127.0.0.1
-              edpm_ovn_metadata_agent_metadata_agent_DEFAULT_metadata_proxy_shared_secret: 12345678
+              edpm_ovn_metadata_agent_DEFAULT_transport_url: $(oc get secret rabbitmq-transport-url-neutron-neutron-transport -o json | jq -r .data.transport_url | base64 -d)
```

Comment:
None of the `dataplane_adoption` role is currently included in the CI playbook (the role is commented out for now), so the job won't tell us. But in theory i think it should work:

```
$ cat <<EOF
> asdf $(echo ghij)
> EOF
asdf ghij
```

The `oc` command will be authenticated the same way as the `oc apply -f <<EOF` that you quoted -- the `{{ oc_header }}` can add `oc` into `$PATH` if it wasn't there already (depending on execution environment), and at the beginning of the test suite we explicitly authenticate, ensuring cached authentication for the rest of the test suite run.

Reply:
I thought we'd see the output of the CI job in order to confirm before merging. But since it's not in CI yet it can't break CI, so we're safe to merge.